NYCJobsWeb/Web.config

```diff
@@ -5,7 +5,7 @@
   -->
 <configuration>
   <appSettings>
-    <add key="BingApiKey" value="[Bing API Key]" />-->
+    <!--<add key="BingApiKey" value="[Bing API Key]" />-->
```

Comment:
It's weird that the key is defined twice, but why remove this one? Our intent here was to force the sample to fail if the person running the sample forgets to put their API key here.